### ISSUE TITLE
Location of typing notification

### ISSUE THREAD

**Issue by jknigga** [telegramdesktop/tdesktop#271]
I would love to see the typing notification either just moved or added to the message window. It would be much easier to see while actually messaging someone rather than having to look in the contact list.

**Comment by telegramdesktop** [comment]
testtest

**Comment by LibertyPaul** [comment]
Test ACK

**Comment by Geobert** [comment]
Agree too, it is already in the header, but when typing, the eyes is focused in the typing field and we can miss that.

**Comment by tdesktop-closer** [comment]
Hey there!

We're automatically closing this issue since there was no activity in this issue since 466 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

<sub>(Please note that this is an [automated](https://github.com/auchri/github_auto_closer) comment.)</sub>

**Comment by Geobert** [comment]
Still relevant, having the typing notification on top while our typing is done at the bottom can lead to missing the information the other one is typing

**Comment by bagonyi** [comment]
I still find this a minor annoyance a year later. Could we have something like Viber has? 
![viber_typing](https://user-images.githubusercontent.com/604444/37398636-0f305588-2777-11e8-9821-d2e9d071ccc9.png)

**Comment by tdesktop-closer** [comment]
Hey there!

We're automatically closing this issue since there was no activity in this issue since 385 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

<sub>(Please note that this is an [automated](https://github.com/auchri/github_auto_closer) comment.)</sub>

**Comment by Geobert** [comment]
This is still relevant

**Comment by Aokromes** [comment]
https://i.imgur.com/AQdWOiD.png this is not enough for you? the way is done matches android way.

**Comment by Geobert** [comment]
Yes it matches Android way, and I also reported that on Android.
Having the typing notification so far from the typing zone is far from ideal. I constantly missing it because I'm focused on the typing zone

**Comment by jknigga** [comment]
@Aokromes - what do you mean that it "matches android way"? In terms of usability, the example that @bagonyi showed makes way more sense than where the notification is currently in Telegram. The typing indicator in the header of the chat is almost impossible to notice, unless you are explicitly looking for it. Having the indicator at the bottom of the chat window, near the text entry (again, as shown in the [example above](https://github.com/telegramdesktop/tdesktop/issues/271#issuecomment-372982879)) is a much better implementation.

**Comment by Geobert** [comment]
@jknigga : @Aokromes means that it also like that on Android

**Comment by Geobert** [comment]
I still stand by this idea

**Comment by Geobert** [comment]
Before a bot comes and auto close it, I still think it's better to have the typing notification at the bottom instead of in the chat's header. And I'd hope Android client to do that as well.
